db/tenant-capability-google-doc.html

```
@@ -0,0 +1,325 @@
+  end
+  subgraph API["REST API"]
+    B[Resolve site set]
+    C[Upsert TenantSiteCapabilityAssociation]
+  end
+  subgraph Data["Persistence"]
+    D[(tenant)]
+    E[(tenant_site_capability_association)]
+    F[(site)]
+    G[(infrastructure_provider)]
+  end
+  A -->|"sites[], capabilityName, infrastructureProviderId, enabled"| B
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)